### PR TITLE
two entities-api-added

### PR DESCRIPTION
**Pull request by MallanagoudaB**

### CODE REVIEW

File: src/controllers/v1/entities.js [hunk 1]
```diff
@@ -606,13 +614,148 @@ module.exports = class Entities extends Abstract {
...
+      * Get the immediate entities .
+      * @method
+      * @name subEntityList
+      * @param  {Request} req request body.
+      * @param {String} req.params._id - entityId
+      * @returns {JSON} Returns list of immediate entities
+     */
+
+    subEntityList(req) {
+
```
Comment (VISHNUDAS-tunerlabs) [inline src/controllers/v1/entities.js:653]: @MallanagoudaB extra line required? formate the code

File: src/controllers/v1/entities.js [hunk 2]
```diff
@@ -606,13 +614,148 @@ module.exports = class Entities extends Abstract {
...
+		  if( !(req.params._id || req.body.entities) ) {
+			return resolve({
+			  status :  HTTP_STATUS_CODE.bad_request.status,
+			  message : constants.apiResponses.ENTITY_ID_NOT_FOUND
+			})
+		  }
+	  
+			try {
+		    // Call 'entitiesHelper.subEntityList' to retrieve sub-entities based on the request parameters
+			  let entityDocuments = await entitiesHelper.subEntityList(
```
Comment (VISHNUDAS-tunerlabs) [inline src/controllers/v1/entities.js:665]: @MallanagoudaB API is tested with pagination and search capability? If not test once

File: src/generics/constants/common.js [hunk 3]
```diff
@@ -22,4 +22,7 @@ module.exports = {
 	GET_METHOD: 'GET',
 	GET_FORM_DATA: '/plugin/v1/form/read',
 	FORM_SERVICE_URL: 'http://player:3000',
+	ENTITYTYPE : "entityType",
+    GROUPS : "groups",
```
Comment (VISHNUDAS-tunerlabs) [inline src/generics/constants/common.js:26]: @MallanagoudaB correct intendation

**Review by VISHNUDAS-tunerlabs — CHANGES_REQUESTED** [review]
reviewed on 9-may 6 AM

File: src/controllers/v1/entities.js (hunk 2 above)
Comment (MallanagoudaB) [reply src/controllers/v1/entities.js:665]: Yes, both in search capability i applied as empty then i was getting some respose when i added number then it was not giving me response so search capability is tested .

**Review by VISHNUDAS-tunerlabs — APPROVED** [review]

**Pull request merged**